packages/component/src/Activity/ActivityFeedback.tsx

```diff
@@ -59,9 +63,15 @@ function ActivityFeedback({ activity }: ActivityFeedbackProps) {
 
   const feedbackActions = useMemo<ReadonlySet<OrgSchemaAction>>(() => {
```

Comment:
The fact that we're not recalculating actions upon selected action is updated bothers me :rofl: 

@lexi-taylor are you saying this causes loops? Have you found why? Why this is safe to avoid recalculating the value in such cases?

Reply:
Unskipped, let's see what happens.